lib/ex_format.ex

```diff
@@ -416,7 +416,15 @@ defmodule ExFormat do
 
   # Tuple containers
   def to_string({:{}, _, args} = ast, fun, state) do
-    tuple = "{" <> tuple_to_string(args, fun, state) <> "}"
+    inside_list? = Map.get(state, :inside_list?)
```

Comment:
Actually `inside_list?` is not always in `state` because I didn't add it as a default attribute when initializing the `state` map. That's why I have to use Map.get/2 which returns a nil if `inside_list?` is not found. Hence this explains why I have to do `inside_list? == true`, and why I didn't use the built-in syntax for map update. :P

Will add it as a default attribute to the `state` map during [initialization](https://github.com/uohzxela/ex_format/blob/ac8163f0a6758ba8cfbd6ffa748ef383e804e0dd/lib/ex_format.ex#L75) if there's no objections.